solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -254,7 +261,7 @@ protected Object applyUpdatesToManagedData(Object updates) {
       }
     }
 
-    return store.modelAsManagedResources();
+    return modelAsManagedResources(store);
```

Comment:
To be consistent between the Managed Model|Feature store, both should have their update() calls consolidated with their respective addFeature|Model functions